Tasks/NuGetAuthenticateV1/main.ts

```diff
@@ -23,14 +23,21 @@ async function main(): Promise<void> {
         const feedUrl = tl.getInput("feedUrl");
         const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
 
+        // Only cross-org feedUrls are supported with Azure Devops service connections. If feedUrl is internal, the task will fail.
         if (feedUrl && entraWifServiceConnectionName) {
             tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName, feedUrl));
-            await configureEntraCredProvider(ProtocolType.NuGet, feedUrl, entraWifServiceConnectionName);
+            await configureEntraCredProvider(ProtocolType.NuGet, entraWifServiceConnectionName, feedUrl);
+            federatedFeedAuthSuccessCount++;    
             console.log(tl.loc("Info_SuccessAddingFederatedFeedAuth", feedUrl));
-            federatedFeedAuthSuccessCount++;
             
             return;
-        }  else if (feedUrl || entraWifServiceConnectionName) {
+        }
+        // If the user doesn't provide a feedUrl, use the Azure Devops service connection to replace the Build Service
+        else if (!feedUrl && entraWifServiceConnectionName) {
+            //configureCredProviderForSameOrganizationFeeds(ProtocolType.NuGet, entraWifServiceConnectionName);
```

Comment:
This needs to be commented out once the new common package version is available. Otherwise it will break the build

Reply:
Removed!